src/Kernel.php

```diff
@@ -19,16 +19,16 @@ class Kernel extends BaseKernel
 
     public function getCacheDir()
     {
-        if (getenv('CACHE_PATH')) {
-            return getenv('CACHE_PATH') . '/' . $this->environment;
+        if (isset($_ENV['CACHE_PATH'])) {
+            return$_ENV['CACHE_PATH'] . '/' . $this->environment;
```

Comment:
If `CACHE_PATH` is left empty, this will attempt to write data to the root `/` directory. Maybe this needs an additional check of emptiness?

```suggestion
            return $_ENV['CACHE_PATH'] . '/' . $this->environment;
```